src/cljs/org/broadinstitute/firecloud_ui/page/method_config_importer.cljs

```diff
@@ -230,15 +230,18 @@
    (fn [{:keys [props state]}]
      [table/Table
       {:columns [{:header "Type" :starting-width 100}
-                 {:header "Namespace" :starting-width 150}
-                 {:header "Name" :starting-width 200 :as-text #(% "name") :sort-by :text
+                 {:header "Name" :starting-width 450 :as-text #(% "name")
```

Comment:
I don't like "Method or Configuration".  It just feels weird.  I'd say go with either "Item" or "ID".